homeassistant/components/sia/strings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "SIA Alarm Systems",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "port": "[%key:common::config_flow::data::port%]",
+          "protocol": "Protocol",
+          "account": "Account ID",
+          "encryption_key": "Encryption Key",
+          "ping_interval": "Ping Interval (min)",
+          "zones": "Number of zones for the account",
+          "additional_account": "Additional accounts"
+        },
+        "title": "Create a connection for SIA based alarm systems."
+      },
+      "additional_account": {
+        "data": {
+          "account": "[%key:component::sia::config::step::user::data::account%]",
+          "encryption_key": "[%key:component::sia::config::step::user::data::encryption_key%]",
+          "ping_interval": "[%key:component::sia::config::step::user::data::ping_interval%]",
+          "zones": "[%key:component::sia::config::step::user::data::zones%]",
+          "additional_account": "[%key:component::sia::config::step::user::data::additional_account%]"
+        },
+        "title": "Add another account to the current port."
+      }
+    },
+    "error": {
+      "invalid_key_format": "The key is not a hex value, please use only 0-9 and A-F.",
+      "invalid_key_length": "The key is not the right length, it has to be 16, 24 or 32 characters hex characters.",
```

Comment:
"Characters" twice.
Reported by Łukasz Ch. in Lokalise ([comment](https://app.lokalise.com/project/130246255a974bd3b5e8a1.51616605/proxy?akey_id=92109656&showcomments=1&search=component%3A%3Asia%3A%3Aconfig%3A%3Aerror%3A%3Ainvalid_key_length))

Reply:
thanks, I'm fixing this is the followup PR here: #51108